copyright_header.txt

```diff
@@ -0,0 +1,4 @@
+“ProMMiS” was produced under the DOE Process Optimization and Modeling for Minerals Sustainability
```

Comment:
```suggestion
“PrOMMiS” was produced under the DOE Process Optimization and Modeling for Minerals Sustainability
```

Reply:
Good catch!  I've updated it and re-ran `addheader` so all the files have changed - again.